java/src/org/openqa/selenium/grid/commands/InfoCommand.java

```diff
@@ -109,7 +112,7 @@ public Executable configure(PrintStream out, PrintStream err, String... args) {
           break;
       }
 
-      String path = getClass().getPackage().getName().replaceAll("\\.", "/") + "/" + toDisplay;
+      String path = getClass().getPackage().getName().replace('.', '/') + "/" + toDisplay;
```

Comment:
Why is this changed? Just curious

Reply:
Similar to the other changes to make the `Pattern` instances static. The `#replaceAll(String, String)` method considered the first argument to be a regex, meaning it needs to be compiled for each call.

But since this call is just matching a literal dot, we can replace it with `#replace(char, char)` which will iterate through the characters in the String and replaces them.